Dashboard/ses_get_emails.py

```diff
@@ -49,14 +32,11 @@ def get_ses_emails(boto_ses_client: ses_client, method: str) -> list[str]:
 `all`, `verified`, `unverified`, you passed %s of type %s', method, type(method))
     return []
 
+def is_ses_verified(ses_client: ses_client, email: str) -> bool:
```

Comment:
Consider changing ses_client variable here or the import but not necessary

Reply:
Good spot! I've changed it now!